src/components/Card.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="box">
+    <div class="media-content">
+      <div class="content">
+        <div class="delete-div">
+          <span>{{filterTask.title}}</span>
+          <button v-on:click="deleteTask(filterTask.id)" class="delete" aria-label="delete"></button>
+        </div>
+      </div>
+    </div>
+    <div class="footer-todo">
+      <span>
+        <!-- <i class="fas fa-pen fa-lg tombol"></i> -->
+      </span>
+      <span v-on:click="moveLeft(filterTask)">
+        <i class="fas fa-caret-square-left fa-lg tombol"></i>
+      </span>
+      <span v-on:click="moveRight(filterTask)">
+        <i class="fas fa-caret-square-right fa-lg tombol"></i>
+      </span>
+    </div>
+  </div>
+</template>
+
+<script>
+import axios from "axios";
+export default {
+  name: "Card",
+  props: ["filterTask", "category"],
+  methods: {
+    getTasks() {
+      this.$emit("getTasks");
+    },
+    deleteTask(taskId) {
+      axios({
+        url: `https://limitless-fjord-07254.herokuapp.com/tasks/delete/${taskId}`,
```

Comment:
axios yang digunakan bisa bikin instancenya jadi nanti kalo butuh ganti baseUrl tinggal ganti satu aja Creating an instance
You can create a new instance of axios with a custom config.

axios.create([config])
const instance = axios.create({
  baseURL: 'https://some-domain.com/api/',
  timeout: 1000,
  headers: {'X-Custom-Header': 'foobar'}
});

Reply:
siap mas